src/js/components/WaitForSettings/WaitForSettings.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import { isActiveZoneOnCloudflare } from '../../selectors/activeZone';
+import Loading from 'cf-component-loading';
+import Text from 'cf-component-text';
+import { FormattedMessage, injectIntl } from 'react-intl';
+
+class WaitForSettings extends Component {
+
+  render() {
+    let activeZone = this.props.activeZone;
+    let analytics = (typeof this.props.analytics != 'undefined' ? this.props.analyics : true);
+    let settings = (typeof this.props.settings != 'undefined' ? this.props.settings : true);
```

Comment:
The reason I didn't do only booleans is because we would have to write 3 more selectors that just check the existence of stuff rather than returning it.